docker/node_cache/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:20.11.0 as node_cache
+ADD ./ReactApp/package.json /ReactApp/package.json
+WORKDIR /ReactApp
+ENV PATH /ReactApp/node_modules/.bin:$PATH
+RUN npm install --legacy-peer-dep
```

Comment:
Do we still need `--legacy-peer-dep`? I do not know whether we removed all the remnants of older MUI.

Reply:
yes unfprtunately